static/templates/left_sidebar.hbs

```diff
@@ -42,46 +42,46 @@
                     <span class="filter-icon">
                         <i class="fa fa-star" aria-hidden="true"></i>
                     </span>
-                    {#- squash whitespace -#}
-                    <span>{{ _('Starred messages') }}</span>
+                    {{!-- squash whitespace --}}
+                    <span>{{t 'Starred messages' }}</span>
                     <span class="unread_count"></span>
                 </a>
                 <span class="arrow starred-messages-sidebar-menu-icon"><i class="zulip-icon zulip-icon-ellipsis-v-solid" aria-hidden="true"></i></span>
             </li>
-            <li class="top_left_recent_topics top_left_row" title="{{ _('Recent topics') }} (t)">
+            <li class="top_left_recent_topics top_left_row" title="{{t 'Recent topics' }} (t)">
                 <a href="#recent_topics">
                     <span class="filter-icon">
                         <i class="fa fa-clock-o" aria-hidden="true"></i>
                     </span>
-                    {#- squash whitespace -#}
-                    <span>{{ _('Recent topics') }}</span>
+                    {{!-- squash whitespace --}}
```

Comment:
The `squash whitespace` comments were not correctly migrated to actually squash whitespace using handlebars whitespace control; I fixed that while merging.